src/store-api/src/metadata.rs

```diff
@@ -666,6 +679,26 @@ impl RegionMetadataBuilder {
         }
     }
 
+    fn change_column_inverted_index_options(
+        &mut self,
+        column_name: String,
+        value: bool,
+    ) -> Result<()> {
+        for column_meta in self.column_metadatas.iter_mut() {
+            if column_meta.column_schema.name == column_name {
+                ensure!(
```

Comment:
discussion: i wonder if `String` is the only data type that support inverted index? what about json or other datatypes.

Reply:
Currently we only support string type. Other datatypes like json may be supported in the future, and we can adjust this correspondingly. See #5080